.github/workflows/wheels-manylinux-test.yml

```diff
@@ -118,7 +118,7 @@ jobs:
       matrix: ${{ fromJSON(needs.wheel-test-compute-matrix.outputs.MATRIX) }}
     runs-on: "linux-${{ matrix.arch }}-gpu-${{ matrix.gpu }}-${{ matrix.driver }}-1"
     container:
-      image: "rapidsai/citestwheel:cuda-devel-${{ matrix.ctk }}-${{ matrix.image }}"
+      image: "rapidsai/citestwheel:cuda${{ matrix.ctk }}-${{ matrix.image }}-py${{ matrix.python }}"
```

Comment:
This is the right change, just noting that this will potentially impact #90 CC @rlratzel